sbi/samplers/vi/vi_quality_control.py

```diff
@@ -114,9 +115,9 @@ def psis_diagnostics(
     M = int(min(N / 5, 3 * np.sqrt(N)))
     with torch.no_grad():
         if proposal is None:
-            samples = q.sample((N,))
+            samples = q.sample(Size((N,)))
         else:
-            samples = proposal.sample((N,))
+            samples = proposal.sample(Size((N,)))
```

Comment:
I also encountered this recently...It's quite annoying imo, I hope that torch loosens their typing requirement here.